canonical-web-and-design/maas-ui
# fix(ui): search repos by display name

## Done

- Add the package repository display name as a search attribute.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit /MAAS/r/settings/repositories.
- Search for "extra". You should get a matching repo.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1783.
